### ISSUE TITLE
TODO: extend AppKit `NSDocumentController` to manage recent items via `applicationd`

### ISSUE THREAD

**Issue by mszoek** [mszoek/airyx#105]
Depends on #104. 
note: don't remove handling of `NSRecentDocumentPaths` in `NSUserPreferences`. That is the per-app recent items list.

**Comment by mszoek** [comment]
No longer needed with WindowServer UI
